troupon/troupon/urls.py

```diff
@@ -5,8 +5,9 @@
 from account import views
 
 urlpatterns = [
-    url(r'^auth/', include('account.urls')),
+	url(r'^auth/', include('account.urls')),
```

Comment:
I think this line should be removed completely. I don't think tests should have been written to test this route.